flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/WindowUtil.scala

```diff
@@ -217,6 +217,10 @@ object WindowUtil {
           null
         }
         val interval = getOperandAsLong(windowCall.operands(2))
+        if (interval <= 0) {
+          throw new TableException(
```

Comment:
since `TableException` is already used for getOperandAsLong
  https://github.com/apache/flink/blob/cd0cfe42c15042a3c27a0c341416aa32c0d594b6/flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/WindowUtil.scala#L390-L394